packages/demand-bloc-track-trip/src/tripTransformer.ts

```diff
@@ -0,0 +1,152 @@
+import {
+  TripFollowResponse as OriginalTripFollowResponse,
+  PassengerDetails,
+  LatLng,
+  TripStatus,
+} from '@karhoo/demand-api'
+import { TripStatusCodes } from './statuses'
+
+const dv = (v: string | undefined) => v || ''
```

Comment:
What does `dv` mean? From this name I can't understand the purpose of this function

Reply:
**d**efault **v**alue - the aim of this name is shortness as it would be used a lot in mapping and a long name will make code heavier.